modules/percolator/src/main/java/org/elasticsearch/percolator/PercolateQueryBuilder.java

```diff
@@ -349,7 +349,7 @@ protected void doXContent(XContentBuilder builder, Params params) throws IOExcep
                 try (XContentParser parser = XContentHelper.createParser(NamedXContentRegistry.EMPTY,
                         LoggingDeprecationHandler.INSTANCE, document)) {
                     parser.nextToken();
-                    XContentHelper.copyCurrentStructure(builder.generator(), parser);
+                    XContent.copyCurrentStructure(builder.generator(), parser);
```

Comment:
can we call `builder.copyCurrentStructure(parser)` instead?

Reply:
Yep, this is done now